scipy/optimize/lbfgsb.py

```diff
@@ -0,0 +1,32 @@
+# This file is not meant for public use and will be removed in SciPy v2.0.0.
+# Use the `scipy.optimize` namespace for importing the functions
+# included below.
+
+import warnings
+from . import _lbfgsb_py
+
+
+__all__ = [  # noqa: F822    'LbfgsInvHessProduct',
```

Comment:
Something is wrong here, `LbfgsInvHessProduct` should be on the next line.

Reply:
Ah! Yes. I think I missed out on pressing an enter here 😅.